lib/checks/shared/focusable-disabled.js

```diff
@@ -0,0 +1,25 @@
+const elementsThatCanBeDisabled = [
+	'BUTTON',
+	'FIELDSET',
+	'INPUT',
+	'SELECT',
+	'TEXTAREA'
+];
+
+const tabbableElements = virtualNode.tabbableElements;
+
+if (!tabbableElements || !tabbableElements.length) {
+	return true;
+}
+
+const relatedNodes = tabbableElements.reduce((out, { actualNode: el }) => {
+	const nodeName = el.nodeName.toUpperCase();
+	// populate nodes that can be disabled
+	if (elementsThatCanBeDisabled.includes(nodeName)) {
+		out.push(el);
+	}
+	return out;
+}, []);
+this.relatedNodes(relatedNodes);
+
+return relatedNodes.length > 0;
```

Comment:
I think this needs to be `relatedNodes.length === 0`, same for the other one. Please verify, but for sure, the following shouldn't pass this rule, and right now it does.

```html
<div aria-hidden="true">
  <a href="">foo</a>
  <button>bar</button>
</div>
```